src/Sulu/Bundle/RouteBundle/Tests/Unit/Generator/SymfonyExpressionTokenProviderTest.php

```diff
@@ -22,6 +22,7 @@ public function testResolve()
     {
         $translator = $this->prophesize(TranslatorInterface::class);
         $translator->getLocale()->willReturn('de');
+        $translator->setLocale('en')->shouldBeCalled();
         $translator->setLocale('de')->shouldBeCalled();
         $entity = new \stdClass();
```

Comment:
maybe use this snippet here:

```
$entity = $this->prophesize(RoutableInterface::class);
$entity->getLocale()->wilLReturn('en');
```

Then you should be able to remove the `is_a` from the production code